test/e2e/regcenter/nacos_test.go

```diff
@@ -0,0 +1,46 @@
+package regcenter_test
+
+import (
+	"e2e/tools"
+	"e2e/tools/common"
+	. "github.com/onsi/ginkgo/v2"
```

Comment:
Let's use `goimport` to solve the imports

Reply:
This is the official recommendation of ginkgo, which can make the unit test code look more elegant

Reply:
```
diff --git test/e2e/regcenter/nacos_test.go test/e2e/regcenter/nacos_test.go
index 250270a..bdd48d0 100644
--- test/e2e/regcenter/nacos_test.go
+++ test/e2e/regcenter/nacos_test.go
@@ -3,9 +3,10 @@ package regcenter_test
 import (
        "e2e/tools"
        "e2e/tools/common"
+       "time"
+
        . "github.com/onsi/ginkgo/v2"
        . "github.com/onsi/gomega"
-       "time"
 )

 var _ = Describe("Nacos", Ordered, func() {
```

Here is the result after sorting by goimport.